level1a/handlers/level1a.py

```diff
@@ -70,6 +72,23 @@ class Level1AException(Exception):
     pass
 
 
+class RetriesExceeded(Exception):
+    pass
+
+
+def s3_backoff(caller: Callable):
+    def wrapper(*args, **kwargs):
```

Comment:
Tror du eventuellt vill lägga till "wraps" för att behålla den dekorerade funktionens signatur.
```suggestion
    @wraps(caller)
    def wrapper(*args, **kwargs):
```


```
from functools import wraps
```